File: src/strategy/c#/Taxer.cs
namespace Strategy
{
    interface ITaxer
    {
        double calculate(Check check);
    }

    class ISS : ITaxer
    {
        double ITaxer.calculate(Check check){
            return check.getTotal() * 0.1;
        }
    }

    class ICMS : ITaxer
    {
        double ITaxer.calculate(Check check){
            return check.getTotal() * 0.06;
        }
    }
}

File: src/strategy/javascript/Taxer.js
var ISS = function() {
    this.calculate = function(check){
        return check.getTotal() * 0.1;
    }
}

var ICMS = function() {
    this.calculate = function(check){
        return check.getTotal() * 0.06;
    }
}

module.exports = {ICMS:ICMS, ISS:ISS}